mypy_django_plugin/main.py

```diff
@@ -186,12 +187,17 @@ def get_function_hook(self, fullname: str) -> Optional[Callable[[FunctionContext
 
             if helpers.is_model_subclass_info(info, self.django_context):
                 return partial(init_create.redefine_and_typecheck_model_init, django_context=self.django_context)
-        return None
+
+        return partial(self.check_function_call, fullname=fullname)
```

Comment:
We cannot do that :(
It will slow down type-checking for all function calls for all users.

Reply:
I know, it's the best option I could find though.. We could e.g. inspect function args and see if there's any abstract model class expected (basically move a part of `check_function_call`) here. Apart from that, I'm out of ideas.

Reply:
We can try adding `is_abstract = True` to `TypeInfo` of the model as well.

Reply:
Yeah, I could try that out.

It's not going to cover for the manager calls. But might be a different story, as Django adds no manager attribute to an abstract model, it might work better to handle that differently.

Reply:
> We can try adding `is_abstract = True` to `TypeInfo` of the model as well.

Right, so I made that happen. But had to make something an abstract attribute. So I opted for `DoesNotExist` and `MultipleObjectsReturned` since those are only added on concrete model classes by the metaclass